### DISCUSSION TITLE
Error when uploading images "This driver does not support creating temporary URLs."

### DISCUSSION THREAD

**Discussion by R1NEJ0** [filamentphp/filament#2317] [Help] [answered]
I'm getting the following error when I upload a image.

This driver does not support creating temporary URLs.

In form I use:
FileUpload::make('foo').

Everything seems ok, I upload the image into folder but when I try to display the table with info I get that message.

Anyone knows why?

Thanks in advance.

**Comment by R1NEJ0** [comment, marked as answer]
I started a new project and works fine in the new project... :S

**Reply by danharrin** [reply]
The issue is when you use a public disk but have previously uploaded files with the incorrect permissions. I've fixed the exception in #2351

**Reply by R1NEJ0** [reply]
Thanks for the great job!
